cmd/kubeops/main.go

```diff
@@ -107,6 +110,30 @@ func main() {
 		negroni.Wrap(http.StripPrefix(assetsvcPrefix, assetsvcProxy)),
 	))
 
+	parsedKubernetessvcURL, err := url.Parse("https://kubernetes.local")
+	if err != nil {
+		log.Fatalf("Unable to parse the Kubernetes SVC URL: %v", err)
+	}
+	kubesvcProxy := httputil.NewSingleHostReverseProxy(parsedKubernetessvcURL)
+	//Add rootCA for certificate validate of self signed certificate
+	caCert, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/ca.crt")
```

Comment:
I see we have hardcoded https://kubernetes.local url above. Should we hardcode the file location also ? Fine if that is how it is been done elsewhere too.